recipe/build.sh

```diff
@@ -11,17 +15,15 @@ configure_args=(
     --without-libresolv
 )
 
+
+autoreconf --force --verbose --install
+if [[ ${target_platform} == "osx-arm64" ]]; then
+    sed -i 's/^\*\-\*\-linux\*/*-*-darwin*/' configure
+    export CFLAGS="${CFLAGS} -DHAVE_DEV_PTMX=1"
+fi
 ./configure "${configure_args[@]}"
 make -j$CPU_COUNT
 make install
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR:-}" != "" ]]; then
 make check
-
```

Comment:
This looks like ancient code - but on a quick glance it seems to at least attempt to do reasonable things (removing the static libraries, la files, documentation) - is that not relevant anymore?

Reply:
I've examined the files locally and there are some static libraries to be deleted, but the names are different now:
```
liblftp-jobs.a
liblftp-jobs.la
liblftp-tasks.a
liblftp-tasks.la
```
I'll make sure these are deleted.

It also appears there is nothing installed by the package into `share/doc`, only `share/man`, which I think should be kept.